packages/create-novu-app/templates/index.ts

```diff
@@ -175,7 +193,9 @@ export const installTemplate = async ({
     version: "0.1.0",
     private: true,
     scripts: {
-      dev: "next dev --port=4000",
+      tunnel: "tsx scripts/tunnel.mts",
+      "next-dev": `next dev --port=${port}`,
+      dev: 'concurrently -k --restart-tries 5 --restart-after 1000 --names "📡 TUNNEL,🖥️  SERVER" -c "bgBlue.bold,bgMagenta.bold" "npm:tunnel" "npm:next-dev"',
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** 😍 love the emojis‏